content/posts/2022-10-08-why-good-code-style-matters.md

```diff
@@ -1,8 +1,10 @@
 ---
 title: Why good code style matters
-date: 2022-10-24T10:10:19.885Z
+date: 2023-10-24T10:10:19.885Z
+
 featuredImage: /post-images/good-style-article-prettier.png
-draft: false
+
+draft: true
 tags:
   - CleanCode
 ---
```

Comment:
Leave some comment around it, like "This code is for testing only DO NOT MERGE IT"
So it doesn't get accidentaly merged